docs/data-access/icgc-25k-data.md

```diff
@@ -46,6 +49,25 @@ Authentication to the server is done using username and password:
 
 You can connect to this server using any SFTP client of your choice.
 
+### AWS details
+
+The AWS bucket is located at: 
+- **Host**: `https://object.genomeinformatics.org`
+
+No additional authentication is required.
+
+To navigate and explore the data:
+```
+aws s3 ls s3://icgc25k-open --endpoint-url https://object.genomeinformatics.org --no-sign-request
```

Comment:
naive question: are people expected to know at this point of the instructions that they need an aws client, etc?

Reply:
We do, as the same is expected for SFTP and filezilla. It was decided at the time, not to endorse.